### ISSUE TITLE
App icon isn't reachable via keyboard navigation

### ISSUE THREAD

**Issue by thomasheartman** [enonic/app-users#647]
The 'Users' app icon in the top left of the nav bar isn't reachable via keyboard. This means that I can't navigate back to the root users page from any of the sub-pages, such as the id provider page, for instance. 

From a little bit of research, I found that the app icon is just a div with event listeners. Ideally, I think this should be an anchor element. It could potentially be argued that it should be a button, but I think an `a` is more appropriate (though I'm happy to be proven wrong 🙋🏾 ).

For clarity, the icon in question is the top left one in this screenshot:

![image](https://user-images.githubusercontent.com/17786332/124268195-81aab800-db39-11eb-9574-570e010aca49.png)
